src/types.ts

```diff
@@ -17,6 +24,25 @@ export type BloomFilter = {
   salt: string;
 };
 
+export type LocalBlocklist = {
+  domainBlocklist: DomainBlocklist;
+  bloomFilter: BloomFilter;
```

Comment:
They are of 2 different types: one has `url` and `hash` and other one is the actual bloom filter. I renamed the actual bloom filter to `bloomFilterObject` since I want to have the property `bloomFilter` from `DomainBlocklist` be same as in the API.
In my last changes I got read from `LocalBlocklist` and persist domain blocklist and bloom filter separately, since we saving the bloom filter to storage every 5 min could lead to spikes in performance.